cmd/res/device.virtual.float.yaml

```diff
@@ -7,7 +7,15 @@ description: "Example of Device-Virtual"
 
 deviceResources:
 -
-  name: "Enable_Randomization"
+  name: "EnableRandomization_Float32"
+  description: "used to decide whether to re-generate a random value"
+  properties:
+    value:
+      { type: "Bool", readWrite: "W", defaultValue: "true" }
```

Comment:
why the type is Bool for Float32?

Reply:
EnableRandomization_Float32 is used to decide whether to re-generate a random float32 value, so it's type is Boolean